### PR TITLE
Assign DB weights to all runtimes

### PR DESCRIPTION
**Pull request by NikVolf**

### CODE REVIEW

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -132,6 +133,13 @@ parameter_types! {
 	pub const Version: RuntimeVersion = VERSION;
 }
 
+parameter_types! {
+	pub const DbWeight: RuntimeDbWeight = RuntimeDbWeight {
```
Comment (rphmeier) [inline runtime/kusama/src/lib.rs:137]: How is this determined?

Comment (NikVolf) [reply runtime/kusama/src/lib.rs:137]: There is a trie benchmark in substrate

`cargo run --release -p node-bench -- trie` 

Those are a values given there are 200k keys in the database

Comment (rphmeier) [reply runtime/kusama/src/lib.rs:137]: Ok, seems arbitrary, but better than nothing

Comment (NikVolf) [reply runtime/kusama/src/lib.rs:137]: > Ok, seems arbitrary, but better than nothing

It can be updated with governance if database grows :)

**Comment by gavofyork** [comment]
Needs resolving.

**Pull request merged**